jsparagus/gen.py

```diff
@@ -29,7 +29,6 @@
 from __future__ import annotations
 
 import io
-import hashlib
```

Comment:
add it back to the module in which `StateAndTransition` went into, as this is needed to have a stable_hash.

Reply:
It's already in parse_table.py, I just failed to delete it from here.